src/pages/blogPost/BlogPost.jsx

```diff
@@ -0,0 +1,62 @@
+import "./BlogPost.css";
+import {useParams} from "react-router-dom";
+import dateFormatter from "../../helpers/dateFormatter.js";
+import { CaretLeft, X } from "@phosphor-icons/react";
+import {useEffect, useState} from "react";
+import axios from "axios";
+
+function BlogPost() {
+    const [postInfo, setPostInfo] = useState({});
```

Comment:
Ik zou gaan voor een default waarde van `undefined`, dan wordt je check of het ingeladen is of niet wat makkelijker
```suggestion
    const [postInfo, setPostInfo] = useState();
```